### DISCUSSION TITLE
How To Retrieve Mining Fees

### DISCUSSION THREAD

**Discussion by tsmith123** [ACINQ/phoenixd#88] [General]
Not sure if I've missed this but is it possible to use the API to get the amount of sats required to pay off the fees? It would be nice to communicate to my users how many more sats they need to receive before the sats start going into their balance. I realise that the mining fee is unpredictable but I assume you know the cost of this once the channel is established?

**Comment by t-bast** [comment]
> I assume you know the cost of this once the channel is established?

I'm not sure what you mean here? The channel has *not* been established at that point, so as you mention, we don't know how much more stats need to be received because it will depend on the feerate at the time where those sats are received.

We could provide a hint ("if you receive that much right now, the channel will be created") but I'm afraid it won't remove any friction, since that estimate may be obsolete by the time the payment comes, which users won't easily understand.

**Reply by pm47** [reply]
Done in #107
